src/dev/src/app/components/tree-view/tree-view-server-side-demo.ts

```diff
@@ -0,0 +1,333 @@
+import { Component, ViewChild } from '@angular/core';
+import {
+  PagingParams,
+  TreeNodeDataRetriever,
+  TreeNodeData,
+  PagedTreeNodeDataRetriever,
+  FuiTreeViewComponent,
+  TreeViewAutoNodeSelector
+} from '@ferui/components';
+
+@Component({
+  template: `
+    <h1 class="mb-4">FerUI Server Side Tree View Demo</h1>
+    <h3>Overview</h3>
```

Comment:
Okay Jon, i added a new dashboard demo page with all the documentation for using the Tree View api. I added all possible @inputs and @outputs along with descriptions and examples for how to use each. Let me know your thoughts. I also separated the examples to the fui-tabs

Reply:
- You need to format your code blocks properly. Use JsBeautify everywhere like we did in the 'demo.component.ts file' : 
`import * as jsBeautify from 'js-beautify';`
Then : 
`codeBlock = jsBeautify.html(sourceCode);`
- Also, for client-side page, the block for " Non Root Tree View" is not well coloured because you have kept ts comments in it. So move your those comments to the typescript tab instead.
- One more thing : You can design the buttons pretty easily by using [bootstrap classes](https://getbootstrap.com/docs/4.0/components/buttons/), so lets design the buttons you have on server-side page please.